### PR TITLE
Added a dozen new hints for the main menu

### PR DESCRIPTION
**Pull request by Inglonias**
#### Summary
Interface "Added 12 more hints to the main menu"

#### Purpose of change
I added some useful advice for new players.

#### Describe the solution
See above

#### Describe alternatives you've considered
N/A

#### Testing
Taken care of by automated GitHub tests.

### CODE REVIEW

File: data/core/tips.json [hunk 1]
```diff
@@ -618,5 +618,65 @@
     "type": "snippet",
     "category": "tip",
     "text": [ "Hope for the best but prepare for the worst.  A backup stash of necessities can save the day." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Pain can slow you down tremendously and reduces your stats. Check your speed in the @ menu." ]
```
Comment (Night-Pryanik) [inline data/core/tips.json:625]:
```suggestion
    "text": [ "Pain can slow you down tremendously and reduces your stats.  Check your speed in the @ menu." ]
```

File: data/core/tips.json [hunk 2]
```diff
@@ -618,5 +618,65 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Winners don't do drugs. They're too valuable. Winners trade drugs for more bullets." ]
```
Comment (Night-Pryanik) [inline data/core/tips.json:630]:
```suggestion
    "text": [ "Winners don't do drugs.  They're too valuable.  Winners trade drugs for more bullets." ]
```

File: data/core/tips.json [hunk 3]
```diff
@@ -618,5 +618,65 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Face masks protect you from smoke and even disease, but make it harder to breathe, affecting stamina." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "The burgler alarm for a store won't go off if you break through the wall." ]
```
Comment (Night-Pryanik) [inline data/core/tips.json:645]:
```suggestion
    "text": [ "The burglar alarm for a store won't go off if you break through the wall." ]
```

**Comment by Zireael07** [comment]
Love the brute force quote :)

**Review by Maleclypse — COMMENTED** [review]
I like them.

File: data/core/tips.json [hunk 4]
```diff
@@ -618,5 +618,65 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "If brute force isn't working, you're not using enough of it." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Sweat is simulated in Cataclysm.  Excessive sweating will dehydrate you quickly." ]
```
Comment (bombasticSlacks) [inline data/core/tips.json:665]:
```suggestion
    "text": [ "Sweat is simulated in Cataclysm.  Excessive sweating will dehydrate you quickly but keep you cool." ]
```

**Pull request merged**